kiwi/bootloader/config/grub2.py

```diff
@@ -64,8 +64,8 @@ def post_init(self, custom_args):
         self.firmware = FirmWare(
             self.xml_state
         )
-
-        self.hybrid_boot = True
+        if arch == 'x86_64':
+            self.hybrid_boot = True
```

Comment:
not yet, there are Intel atom systems at 32bit which we still want to support... oh well